app.js

```diff
@@ -1,151 +0,0 @@
-document.addEventListener("DOMContentLoaded", () => {
-  let comments = [];
-  // Target DOM Elements
-  let comment_box = document.querySelector("#comment_box");
-  let comment_btn = document.querySelector("#add_comment");
-  let comment_wrapper = document.querySelector("#comment-listing");
-  let reply_textbox = document.querySelector("#reply_comment");
-  let nestedCommentForm = document.querySelector(
-    "#comment_list_inner .comment-area"
-  );
-
-  //initialize loacl stroge if not
-  if (localStorage.getItem("comments") === null)
-    localStorage.setItem("comments", JSON.stringify(comments));
-
-  //Add comment object in localStorge
-  let addComment = (comment) => {
-    let all_comments = getAllComments();
-    all_comments.push(comment);
-    updateStroge(all_comments);
-  };
-
-  //update local stroge with new data
-  let updateStroge = (newData) => {
-    localStorage.setItem("comments", JSON.stringify(newData));
-    renderComments();
-  };
-
-  //Add reply on the same index
-  let addReply = (reply, index) => {
-    let all_comments = getAllComments();
-    all_comments.forEach((element, i) => {
-      if (i == index) {
-        if (element.children) {
-          element.children.push(reply);
-        } else {
-          element.children = [];
-          element.children.push(reply);
```

Comment:
This can be refactored as just one line as below
```js
element.children = [reply];	
```

Reply:
single line was not working it was working only 1 time and later it was not pusing new obj so I did this 

if (i == index) {
        if (element.children) {
          element.children.push(reply);
        } else {
          element.children = [reply];
        }
      }

Reply:
This is what I meant. This is fine.